src/kw/kw.cpp

```diff
@@ -53,6 +70,7 @@ bool KanoWorld::login(string username, string password, bool verbose)
         curl_easy_setopt(curl, CURLOPT_COPYPOSTFIELDS, payload.c_str());
 
         curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, write_function);
+        curl_easy_setopt(curl, CURLOPT_WRITEDATA, this);
```

Comment:
Perhaps something for a later time, but it would be nice if we could use `HTTPClient` @tombettany wrote on top of `Poco`

Reply:
Yep - discussed and agreed